### PR TITLE
Dev 6.1.0

### PR DESCRIPTION
**Pull request by nhunzaker**
`Microcosm.pull` lets you quickly retrieve nested keys

### CODE REVIEW

File: src/getIn.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * getIn
+ * Enumerate through a list of keys to get a value
+ */
+
+export default function getIn (obj, key) {
+  let val  = obj
```
Comment (greypants) [inline src/getIn.js]: y the name reassignment?

**Comment by greypants** [comment]
:+1:

File: src/getIn.js (hunk 1 above)
Comment (nhunzaker) [inline src/getIn.js]: I don't even know. Good idea

Comment (nhunzaker) [inline src/getIn.js]: Done

**Pull request merged**